lib/models/task.dart

```diff
@@ -1,27 +1,33 @@
-import 'package:meta/meta.dart';
+import 'package:flutter/material.dart';
 import 'package:json_annotation/json_annotation.dart';
 import 'package:vikunja_app/components/date_extension.dart';
 
 import 'package:vikunja_app/models/label.dart';
 import 'package:vikunja_app/models/user.dart';
+import 'package:vikunja_app/models/taskAttachment.dart';
 
 @JsonSerializable()
 class Task {
-  int id, parentTaskId, priority, listId;
+  int id, parentTaskId, priority, listId, bucketId;
   DateTime created, updated, dueDate, startDate, endDate;
   List<DateTime> reminderDates;
-  String title, description;
+  String title, description, identifier;
   bool done;
+  Color color;
+  double kanbanPosition;
   User createdBy;
   Duration repeatAfter;
   List<Task> subtasks;
   List<Label> labels;
+  List<TaskAttachment> attachments;
   bool loading = false;
+  // TODO: add position(?)
```

Comment:
The position is only used in list views, it's fine to add this later.

Reply:
This seems to be a pretty new feature in Vikunja? My build from source as of 7/31 let's you rearrange the tasks but doesn't actually save the positions. But I did just try it on the demo version and it worked. Anyway that's something to update in the list view.

Reply:
It's a few months old actually. Are you using a recent frontend + api build?

Reply:
Yes, both built and updated last Sunday. Using sqlite db, running directly in a Ubuntu 22.04 lxc container on a Proxmox host.